competition-simulator.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+![]({{ site.baseurl }}/resources/competition-simulator/arena.png)
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is different to the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulation is based in [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.5. Additional external libraries are not supported.
+
+### Installing the simulation
+
+1. [Download the simulation](), and unzip it somewhere on your computer.
+2. Using the Webots IDE, open the `worlds/Arena.wbt` file.
+
+You may receive a warning about your computer's GPU not being good enough, which can be ignored.
+
+#### Changing your version of Python
+
+If webots is picking up the incorrect version of Python, you'll need to change it. This can be done using `Tools > Preferences > General > Python command`. You'll need to ensure a matching version of Python is installed.
+
+## Overview
+
+Within the IDE, there are a few different panels:
+
+- In the centre of your screen is the 3D simulated view of the arena
+- On the left is a tree hierarchy of all elements in this "world"
+- On the right is your text editor for modifying the `sr_controller` code for example
+- At the bottom is your console
+- At the top are your general controls which include the time controls. Press the centre play button to run the simulation at normal speed.
+
+**Important:** Changes to the world must happen with the simulation paused at 0:00. If e.g. you move an object at a different time, rewinding back to the start will delete your changes.
+
+### Useful links
+
+- [Camera Controls](https://www.cyberbotics.com/doc/guide/the-3d-window#navigation-in-the-scene)
+- [Graphics settings](https://www.cyberbotics.com/doc/guide/preferences#opengl) (Useful if Webots is running slowly)
+
+## Developing your code
+
+The code for your robot is `controllers/sr_controller/sr_controller.py`. An example skeleton code is provided for convenience.
+
+<div class="warning">
+  Please refrain from editing any code in the <code>sr</code> module directory. This contains the code which maps the SR API onto the webots API.
+  Only your controller code will be present in the competition environment.
+</div>
+
+### Updates
+
+Occasionally, we may release an update to the simulation. To update, you will need to copy your code out of the `competition-simulator-master` directory, delete the directory, and re-download it using the above link. Then you can replace the new `controllers/sr_controller/sr_controller.py` with your existing code.
```

Comment:
Do we want to recommend version control systems so teams can share code?

Reply:
I'll put in a warning about backing up code